tests/testcase.coffee

```diff
@@ -25,11 +25,22 @@ class TestCase
     keyBindings = new KeyBindings @settings
     @keyhandler = new KeyHandler @view, keyBindings
     @register = @view.register
+    @name = options.name || "an anonymous test"
+    unless callback then throw "No callback in test"
+    if it?
```

Comment:
Not so much so we can run without mocha, as so we can switch from mocha to something else more easily.